src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs

```diff
@@ -292,6 +292,8 @@ public async Task<ActionResult> WhenDoYouWantToView(int? choice, string agreemen
         public async Task<ActionResult> ViewAllAgreements(string hashedAccountId, string accountLegalEntityHashedId)
         {
             var model = await _orchestrator.GetOrganisationAgreements(accountLegalEntityHashedId);
+            ViewBag.AgreementId = accountLegalEntityHashedId;
```

Comment:
would it be better to have the "AgreementId" on the model rather than using viewbag?